tasks/libs/common/worktree.py

```diff
@@ -73,12 +79,12 @@ def exit_env():
 
 
 @contextmanager
-def agent_context(ctx, branch: str | None, no_checkout=False):
-    """Applies code to the worktree environment if the version is not None.
+def agent_context(ctx, branch: str | None, skip_checkout=False):
+    """Applies code to the worktree environment if the branch is not None.
 
     Args:
         branch: The branch to switch to.
-        no_checkout: If True, the branch will not be switched (no pull will be performed too).
+        skip_checkout: If True, the branch will not be switched (no pull will be performed too).
```

Comment:
nit: might make more sense you use checked out:
```suggestion
        skip_checkout: If True, the branch will not be checked out (no pull will be performed too).
```